azure-proto-compute/Extensions/ResourceGroupExtensions.cs

```diff
@@ -3,58 +3,114 @@
 namespace azure_proto_compute
 {
     /// <summary>
-    /// Naming of these methods
+    /// A class to add extension methods to ResourceGroup.
     /// </summary>
     public static class ResourceGroupExtensions
     {
         #region VirtualMachines
-        public static VirtualMachineOperations VirtualMachine(this ResourceGroupOperations resourceGroup, string name)
+        /// <summary>
+        /// Gets the VirtualMachineOperations for a VirtualMachine.
```

Comment:
I think it might work out well to use the same phrasing we do on the class description for VirtualMachineOperations just put a Gets in front of it

https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#standard-class-documentation

so here it would read
```suggestion
        /// Gets an object representing the operations that can be performed over a specific VirtualMachine.
```